.github/workflows/hackage.yml

```diff
@@ -25,7 +25,8 @@ jobs:
                   "hls-module-name-plugin", "hls-pragmas-plugin",
                   "hls-refine-imports-plugin", "hls-retrie-plugin",
                   "hls-splice-plugin", "hls-tactics-plugin",
-                  "hls-call-hierarchy-plugin", "hls-alternate-number-format-plugin",
+                  "hls-call-hierarchy-plugin",
+                  # "hls-alternate-number-format-plugin",
```

Comment:
Turning it off as a build dep shouldn't stop us uploading to hackage, though?

Reply:
hmm not for the default build but it would not allow to do a pure hackage build forcing the flag to include the plugin

not sure if it worths upload the package to hackage though

Reply:
finally i included it in the hackage build, we have to be strict and avoid possible build errors